storybook-testing/visual-tests.test.js

```diff
@@ -10,12 +10,12 @@ global.window = { ...global };
 
 const getMatchOptions = ({ context }) => {
     return {
-        failureThreshold: 0.2,
+        failureThreshold: 0.1,
         failureThresholdType: 'percent',
         customSnapshotIdentifier: () => context.name.replace(/\s/g, ''),
         // Will output base64 string of a diff image to console in case of failed tests (in addition to creating a diff image).
         // This string can be copy-pasted to a browser address string to preview the diff for a failed test.
-        dumpDiffToConsole: true,
+        dumpDiffToConsole: false,
```

Comment:
This should be true - otherwise you cannot figure see diffs from travis. You can use an environment variable from travis to toggle it if you don't want it enabled locally.

Reply:
It has been changed :) I'm just not fan of it, because travis won't handle more than 3000 lines in job.

Reply:
How else would you debug a broken image in a travis build though? We had to disable ssh'ing into the build. I agree it's ugly as hell and would prefer it off 😆